tests/phpunit/Unit/Maintenance/GlossaryCacheRebuilderTest.php

```diff
@@ -19,8 +19,9 @@
  * @since 1.1
  *
  * @author mwjames
+ * @todo Should be rewritten
```

Comment:
how about skipping instead of removing to keep it as a boilerplate for someone refactoring this?